test/IRGen/conditional_conformance_subclass.swift

```diff
@@ -0,0 +1,153 @@
+func takes_p1<T: P1>(_: T.Type) {}
```

Comment:
Missing RUN line

Reply:
Nah... It's only necessary if you want to run the test.

Reply:
Yeah, it's easier to get all tests passing this way. (Thanks.)